attack.py

```diff
@@ -1,11 +1,10 @@
 # -*- coding: utf-8 -*-
 from argparse import ArgumentParser
 from concurrent.futures import ThreadPoolExecutor, as_completed
-from loguru import logger
+# from loguru import logger
 from os import system

```

Comment:
I commended unused import. I didn't delete it because maybe you use it for debugging.